### ISSUE TITLE
can not show in each level dir

### ISSUE THREAD

**Issue by daixiang0** [oupala/apaxy#95]
```
131 <Directory "/var/www/html">
132     #
133     # Possible values for the Options directive are "None", "All",
134     # or any combination of:
135     #   Indexes Includes FollowSymLinks SymLinksifOwnerMatch ExecCGI MultiViews
136     #
137     # Note that "MultiViews" must be named *explicitly* --- "Options All"
138     # doesn't give it to you.
139     #
140     # The Options directive is both complicated and important.  Please see
141     # http://httpd.apache.org/docs/2.4/mod/core.html#options
142     # for more information.
143     #
144     Options All MultiViews
145 
146     #
147     # AllowOverride controls what directives may be placed in .htaccess files.
148     # It can be "All", "None", or any combination of the keywords:
149     #   Options FileInfo AuthConfig Limit
150     #
151     AllowOverride All
152     #
153     # Controls who can get stuff from this server.
154     #
155     Require all granted
156 </Directory>
```
Then i try to test each level dirs and find only the first dir can have style.
I find a way to slove it is that cp theme dir to each level dir.
Is there any better way to slove it?

**Comment by daixiang0** [comment]
duplicate as https://github.com/AdamWhitcroft/Apaxy/issues/62
I think could enable this change in this repo if possible?

**Comment by Pyroglyph** [comment]
I'm having the same issue. I assume it's a design flaw? To reiterate what daixiang0 said, only the directory in which /themes/ is in will be styled, but not it's subdirectories. Is there an elegant solution that does not require us to copy the /theme/ and .htaccess files to every directory we want styled?

**Comment by rivermont** [comment]
As far as I can tell the only way is two copy the files into each directory. You can save some space if you know what the contents will be by deleting unnecessary images and customizing the `.htaccess` files.

**Comment by Romanmir** [comment]
I just tested on my server, and it looks like you can also ln -s ../theme theme (assuming you are currently in a child directory, you may need to season to taste.)

**Comment by rivermont** [comment]
I was experimenting, and discovered a thing:

~~If you have `theme/` anywhere on you're server, you can replace all occurrences of `theme/` in the `.htaccess` with `/path/to/theme/folder/`. Note the `/` at the start, indicating the server home directory.
This also means that the header, footer, and icons will be updated across-site when they are edited in the one folder.~~

UPDATE:
The `.htaccess` is only needed in the top-level directory (of the directory wanted to be open), but **both** `apache2.conf` and the config file for your server in `sites-enabled/` *must* have the line

> Options ... Indexes ...

For this to work. The styling will then be applied to all subdirectories.

Unless there are any more problems, @AdamWhitcroft I believe this can be closed.

**Comment by GodsVictory** [comment]
Simply replacing `theme/` with `/theme/` worked for me.

**Comment by oupala** [comment]
@GodsVictory had the right answer.

Issue has been fixed in master branch, and in 1.0.0 release.

I'm closing this issue.
